### PR TITLE
docs: add Angular 12 support information

### LINKED ISSUE #42111: Angular 12 support dates missing on angular.io
# Docs or angular.io Bug Report

### Description

Please update the release support policy and schedule section (https://angular.io/guide/releases#support-policy-and-schedule) on angular.io to list support dates for Angular 12. 

### What's the affected URL?

https://angular.io/guide/releases#support-policy-and-schedule

### Expected vs Actual Behavior

Per the blog announcement (https://blog.angular.io/angular-v12-is-now-available-32ed51fbfd49), Angular 12 was released on May 12, 2021. However, the support dates for this version are not listed on angular.io.

### PR DESCRIPTION
**Pull request by iRealNirmal**
Added angular 12 suppport information in release document

Closes #42111

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Document is missing angular 12 support information

Issue Number: #42111 

## What is the new behavior?
Angular 12 support dates are added

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by petebacondarwin — APPROVED** [review]
LGTM!
Revewied-for: global-docs-approvers

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ The following table provides the status for Angular versions under support.
 
 Version | Status | Released     | Active Ends  | LTS Ends
 ------- | ------ | ------------ | ------------ | ------------
+^12.0.0 | Active | May 12, 2021 | Nov 12, 2021 | Nov 12, 2022
 ^11.0.0 | Active | Nov 11, 2020 | May 11, 2021 | May 11, 2022
```
Comment (JoostK) [inline aio/content/guide/releases.md:111]: Can this be changed to LTS?

Comment (iRealNirmal) [reply aio/content/guide/releases.md]: done

**Review by Splaktar — APPROVED** [review]
LGTM, thank you!

I triggered a rerun of the failing `legacy-unit-tests-saucelabs` test that timed out.

**Review by gkalpak — APPROVED** [review]
Thx, @iRealNirmal ✨

Reviewed-for: global-docs-approvers

**Pull request closed without merging**